extensions/vertx/deployment/src/main/java/io/quarkus/vertx/deployment/WebVertxBuildItem.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.vertx.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.runtime.RuntimeValue;
+import io.vertx.core.Vertx;
+
+/**
+ * TODO: DELETE THIS CLASS
```

Comment:
Do we have a follow-up issue? I'm afraid a TODO is not enough and it should be really just a temporary workaround...

Reply:
Well, the issue is well known and being discussed in many places, I doubt it will get lost.